backend/lib/endpoints/posts.js

```diff
@@ -480,10 +486,10 @@ async function routes(app) {
             },
           },
           {
-            $skip: skip || 0,
+            $skip: parseInt(skip) || 0,
           },
           {
-            $limit: limit || COMMENT_PAGE_SIZE,
+            $limit: parseInt(limit) || COMMENT_PAGE_SIZE,
```

Comment:
Weird the `getCommentsSchema` should be validating the querystring and transforming these to ints - but it's doing nothing at all (where as `getPostsSchema` works fine)`. Something to look into later I guess because this works.

Reply:
Yeah, I found that while testing also. Not sure why the schema validation seems to be different between those two endpoints.